src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -70,9 +70,14 @@ def create_branched_package
     end
 
     begin
+      # Service running on package avoids branching it
+      Backend::Api::Sources::Package.wait_service(source_project_name, source_package_name)
+
       BranchPackage.new({ project: source_project_name, package: source_package_name,
                           target_project: target_project_name,
                           target_package: target_package_name }).branch
+
+      Backend::Api::Sources::Package.runservice(source_project_name, source_package_name)
```

Comment:
Branching a package creates files which triggers services. No need to do this manually...

Reply:
Ah! I missed it triggers the source ones. Thanks for the hint. :+1: